apps/prairielearn/src/server.js

```diff
@@ -2025,6 +2025,21 @@ export async function initExpress() {
       coreElements: false,
     }),
   );
+  app.use(
+    '/pl/public/course_instance/:course_instance_id(\\d+)/assessment/:assessment_id(\\d+)/questions',
```

Comment:
Assessments are located in course instances, and this is a similar route to the current ones looking at assessments and assessment questions. I don't think it would make sense to have a redundant course_id portion of the url because we can get the course_id(if needed) using the course_instance_id

Reply:
Fair enough, this is at least consistent with how we structure URLs for course admins!